src/components/LocationAutocomplete/LocationAutocomplete.tsx

```diff
@@ -101,7 +104,14 @@ const LocationAutocomplete: React.FC<Props> = (props) => {
                     }
 
                     if (ACCEPTABLE_API_STATUSES.includes(status)) {
-                        setSuggestionsList(predictions);
+                        if (citiesOnly) {
+                            const result = predictions.filter(
+                                (pred) => !pred.description.toLowerCase().includes('airport')
+                            );
+                            setSuggestionsList(result);
+                        } else {
+                            setSuggestionsList(predictions);
+                        }
```

Comment:
Google considers airports type as cities, so maybe it's true? :) I'd keep airports because the solution doesn't look robust, for example, if we change the language. And, we need to set 2 props for having the cities in the result so that can cause bugs.